packages/create-atomic/templates/src/index.js

```diff
@@ -0,0 +1,25 @@
+async function renewAccessToken() {
```

Comment:
Should I add TS to the generated project right now? It will be necessary in the next phases.